OSD.adoc

```diff
@@ -1,14 +1,16 @@
 ---
-layout: OSD-2022
-title: Open Source Day 2022
-permalink: /OSD-2022/
+layout: OSD
+title: Open Source Day 2023
+permalink: /OSD/
 image:
-  path: https://wildfly-security.github.io/wildfly-elytron/assets/images/posts/OpenSourceDayProject2022.png
+  path: https://prarthonapaul.github.io/wildfly-elytron/assets/images/posts/OSD2023Banner.png
 ---
 :page-liquid:
 
+Welcome, Open Source Day participants! We are glad you have joined us!
 
-Welcome Open Source Day participants! We're glad to have you join us for the hackathon.
+[.banner]
+image::https://prarthonapaul.github.io/wildfly-elytron/assets/images/posts/OSD2023Banner.png[align="left"]
```

Comment:
Same here.

Reply:
I think this comment about tweaking this sentence still needs to be addressed.

Reply:
fixed!